---
name: Ege
surname: Ulgen
position: "MD, PhD"
address: "Genomics England Limited"
phone: +44 7796 427839
pronouns: he/him
profilepic: ../images/eu_pp.jpg
www: egeulgen.github.io/portfolio
email: "Ege.Ulgen@genomicsengland.co.uk"
github: egeulgen
linkedin: ege-ulgen
date: "`r format(Sys.time(), '%B %Y')`"
aboutme: "MD-PhD professional in bioinformatics with a medical degree from Acibadem University in 2015, postdoctoral experience at Yale University in 2016, followed by a PhD in Biostatistics and Bioinformatics in 2022. Currently a Bioinformatics Engineer at Genomics England, where I focus on creating and enhancing software for the cancer genomic medicine service."
output: vitae::twentyseconds
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
library(vitae)
```

# Education

```{r}
library(tibble)
tribble(
  ~ Degree, ~ Year, ~ Institution,
  "Ph.D., Biostatistics and Bioinformatics", "2016-2022", "Acibadem University",
  "M.D., summa cum laude", "2009-2015", "Acibadem University",
) %>% 
  detailed_entries(Degree, Year, Institution)
```

# Experience

```{r}
tribble(
  ~ Experience, ~ Year, ~ Institution, ~Description,
  "Bioinformatics Engineer", "2023-", "Genomics England", "Developing features for and maintaining software for the cancer genomic medicine service.",
  "Lecturer", "2021-2023", "Acibadem University", "Worked as a lecturer for the undergrad and grad courses on biostatistics and bioinformatics topics.",
  "Bioinformatics Specialist", "2018-2023", "Epigenetiks", "Worked on developing and maintaining multiple whole exome sequencing-based analysis pipelines (cancer and rare disease applications) for the Istanbul-based medical diagnostics company.",
  "Postdoctoral Associate", "2016", "Yale University", "Briefly worked as a post-doc associate at the Gunel Lab, Yale School of Medicine, where I had previously done multiple internships between 2011-2014. Research in neurogenetics, particularly malformations of cortical development and brain tumours, performing functional in-vitro analyses to investigate the effect of a SNP in the pathogenesis of gliomas.",
  "Bioinformatics Consultant", "2015-2023", "NeuroOncology Technologies", "Developed and maintained the Whole Exome Sequencing pipeline for the local medical diagnostics company based in Istanbul-Turkey. The pipeline allows for the identification of clinically-relevant genomics findings in brain tumours, especially regarding gliomas",
  "Research Group Member", "2012-", "ACU Brain tumour Research Group", "Participate in research on brain tumours, particularly on gliomas. I assist in biostatistical as well as bioinformatics analyses, focusing on the integration of molecular biological and clinical data to improve diagnosis, prognosis and treatment of glioma.",
) %>% 
  detailed_entries(Experience, Year, Description, Institution)
```

# Skills

## Programming

Python \hfill ![](../images/rating3.png)

R \hfill ![](../images/rating3.png) 

bash \hfill ![](../images/rating2.png)

## Development

git \hfill ![](../images/rating2.png)

docker \hfill ![](../images/rating1.png) 

GitLab CI/CD \hfill ![](../images/rating1.png)

GitHub Actions \hfill ![](../images/rating1.png)

## Workflow Management

nextflow \hfill ![](../images/rating2.png)

snakemake \hfill ![](../images/rating2.png)

# Strengths

**Curiosity** \hfill Proactively seeks innovative solutions.

**Time Management** \hfill Effectively prioritizes tasks, ensuring timely completion of milestones.

**Quick Learner** \hfill Adapts to new challenges swiftly.

# Software

```{r rpkgs}
tmp <- as.data.frame(pkgsearch::ps("Ulgen",size = 100) %>%
  filter(maintainer_name == "Ege Ulgen"))

tmp %>%
  brief_entries(
    what = package, 
    when = as.character(version), 
    with = title
  )
```

# Certification

```{r}
tribble(
  ~Year, ~Type, ~Desc,
  2016, "Machine Learning", "Grade: 94.8% – coursera.org/verify/7QETTCDWHXHP",
  2018, "Finding Hidden Messages in DNA (Bioinformatics I) (with Honors)", "Grade: 96.0% – coursera.org/verify/B5Y2QZ4AWTCL",
  2019, "Genome Sequencing (Bioinformatics II) (with Honors)", "Grade: 99.0% – coursera.org/verify/242DMGMNGFXA",
  2019, "Comparing Genes, Proteins, and Genomes (Bioinformatics III) (with Honors)", "Grade: 100% – coursera.org/verify/N3B77QC9N7V6",
  2019, "Molecular Evolution (Bioinformatics IV) (with Honors)", "Grade: 100% – coursera.org/verify/TM3MWQPRPS47",
  2019, "Genomic Data Science and Clustering (Bioinformatics V) (with Honors)", "Grade: 94.0% – coursera.org/verify/AGX8F2V8PU3G",
  2019, "Bioinformatics Capstone: Big Data in Biology", "Grade: 100% – coursera.org/verify/9AKDLPVN9M2U",
  2019, "Bioinformatics Specialization", "coursera.org/verify/35KWGTCVTJX8"
) %>% 
  brief_entries(
    glue::glue("Certificate in {Type}"),
    Year, 
    Desc
  )
```

# Publications

```{r}
library(dplyr)
scholar::get_publications("5bN2xIYAAAAJ&hl") %>% 
  # filter(cites > 30) %>% 
  distinct(title, .keep_all = TRUE) %>%
  transmute(bibtype = "Article", author = as.character(author),
            title = as.character(title),
            journaltitle = as.character(journal), year, key = row_number()) %>% 
  RefManageR::as.BibEntry() %>% 
  RefManageR::WriteBib("egeulgen.bib")

bib_entries <- bibliography_entries("egeulgen.bib")
bib_entries <- bib_entries[order(bib_entries$issued, decreasing = TRUE), ]
bib_entries
```

# References

Available upon request

