---
name: Ege
surname: Ulgen
position: "MD, PhD"
address: "Genomics England Limited"
phone: +44 7796 427839
pronouns: he/him
profilepic: photo.jpeg
email: "Ege.Ulgen@genomicsengland.co.uk"
github: egeulgen
linkedin: ege-ulgen-378984159
date: "`r format(Sys.time(), '%B %Y')`"
aboutme: "MD-PhD professional in bioinformatics with a medical degree from Acibadem University in 2015, postdoctoral experience at Yale University in 2016, followed by a PhD in Biostatistics and Bioinformatics in 2022. Currently a Bioinformatics Engineer at Genomics England, where I focus on creating and enhancing software for the cancer genomic medicine service."
output: vitae::twentyseconds
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
library(vitae)
```

# Education

```{r}
library(tibble)
tribble(
  ~ Degree, ~ Year, ~ Institution, ~ Where,
  "M.D., summa cum laude", "2009-2015", "Acibadem University", "Istanbul, Turkey",
  "Ph.D., Biostatistics and Bioinformatics", "2016-2022", "Acibadem University", "Istanbul, Turkey",
) %>% 
  detailed_entries(Degree, Year, Institution, Where)
```

# Experience

> **Genomics England Limited** \hfill 2023--present

Working as a Bioinformatics Engineer to develop features for and maintain the cancer short read sequencing pipeline. First Level Manager: Javier Lopez

> **Epigenetiks Genetik Biyoinformatik Yazılım A.Ş** \hfill 2018--2022

Worked on multiple -omics analysis pipelines for the local medical diagnostics company based in Istanbul--Turkey. Responsibilities included: implementation, continuous development and maintenance of whole exome sequencing analysis pipeline for aiding the diagnosis of undiagnosed subjects with rare diseases, and whole exome sequencing analysis pipeline for cancer diagnostics and prognostics. Supervisor: Prof. Ugur Sezerman, Ph.D.

> **Akalin Lab, Max--Delbrück--Centrum für Molekulare Medizin** \hfill 2018

Worked on a project on the effects of "doublet cells" on single--cell RNAseq analysis. Using a real data set and simulations, I demonstrated that doublets cause pseudo--paths (false patterns of differentiation), negatively affect discrimination between different cell types and have a negative effect on imputation results. Supervisors: Vedran Franke, Ph.D., Altuna Akalin, Ph.D.

> **Sezerman Lab, Acibadem University** \hfill 2016--2022

Research in -omics analyses of various complex diseases, including cancers, neurological diseases and gynecological diseases, particularly focused on transriptomics and proteomics analyses, utilizing differential expression analysis tools as well as machine learning approaches to classify patients from controls and identify genes important for pathogenesis. Supervisor: Prof. Ugur Sezerman, Ph.D.

> **NOT Nöroonkoloji Teknolojileri Ltd Şti.** \hfill 2015--2022

Developed and maintained the Whole Exome Sequencing pipeline for the local medical diagnostics company based in Istanbul-Turkey. The pipeline allows for the identification of clinically--relevant genomics findings in brain tumors, especially regarding gliomas. Supervisor: Prof. Koray Ozduman, MD

> **Elective Clerkship as Research Intern at Gunel Lab -- Yale University School of Medicine** \hfill 2014

Research on gliomas. I performed in-vitro experiments, including cloning, cell--culture and luciferase assays to reveal that a risk allele augments the effect of an enhancer targeting the oncogene MYC, hence facilitating gliomagenesis. Prof. Murat Gunel, MD

> **Research Internships at Gunel Lab -- Yale University School of Medicine** \hfill 2011-2013

Research in neurogenetics, particularly malformations of cortical development, utilizing molecular techniques to investigate the underlying genetic/molecular mechanisms, analyzing Whole Exome Sequencing results and verifying relevant findings. Research in vestibular schwannomas, performing functional in-vitro analyses to investigate the effect of a mutation in the pathogenesis of vestibular schwannomas. Supervisor: Prof. Murat Gunel, MD

> **Acıbadem University Brain Tumor Research Group** \hfill 2012--present

Participate in research on brain tumors, particularly on gliomas. I assist in biostatistical as well as bioinformatics analyses, focusing on the integration of molecular biological and clinical data to improve diagnosis, prognosis and treatment of glioma. Supervisor: Prof. Koray Ozduman, MD

# Skills

## Programming

python - 5/5 \hfill R - 5/5 \hfill bash - 4/5 \hfill C++ - 2/5

## Development

git - 4/5 \hfill Docker - 3/5 \hfill GitLab CI - 4/5 \hfill GitHub Actions - 4/5 \hfill PyCharm - 5/5

# Strengths

**Curiosity** \hfill Proactively seeks innovative solutions.
**Time Management** \hfill Effectively prioritizes tasks, ensuring timely completion of milestones.
**Quick Learner** \hfill Adapts to new challenges swiftly.

# Software

```{r rpkgs}
tmp <- as.data.frame(pkgsearch::ps("Ulgen",size = 100) %>%
  filter(maintainer_name == "Ege Ulgen"))

tmp %>%
  brief_entries(
    what = package, 
    when = as.character(version), 
    with = title
  )
```

# Certification

```{r}
tribble(
  ~Year, ~Type, ~Desc,
  2016, "Machine Learning", "Grade: 94.8% – coursera.org/verify/7QETTCDWHXHP",
  2018, "Finding Hidden Messages in DNA (Bioinformatics I) (with Honors)", "Grade: 96.0% – coursera.org/verify/B5Y2QZ4AWTCL",
  2019, "Genome Sequencing (Bioinformatics II) (with Honors)", "Grade: 99.0% – coursera.org/verify/242DMGMNGFXA",
  2019, "Comparing Genes, Proteins, and Genomes (Bioinformatics III) (with Honors)", "Grade: 100% – coursera.org/verify/N3B77QC9N7V6",
  2019, "Molecular Evolution (Bioinformatics IV) (with Honors)", "Grade: 100% – coursera.org/verify/TM3MWQPRPS47",
  2019, "Genomic Data Science and Clustering (Bioinformatics V) (with Honors)", "Grade: 94.0% – coursera.org/verify/AGX8F2V8PU3G",
  2019, "Bioinformatics Capstone: Big Data in Biology", "Grade: 100% – coursera.org/verify/9AKDLPVN9M2U",
  2019, "Bioinformatics Specialization", "coursera.org/verify/35KWGTCVTJX8"
) %>% 
  brief_entries(
    glue::glue("Certificate in {Type}"),
    Year, 
    Desc
  )
```

# Publications

```{r}
library(dplyr)
scholar::get_publications("5bN2xIYAAAAJ&hl") %>% 
  # filter(cites > 30) %>% 
  distinct(title, .keep_all = TRUE) %>%
  transmute(bibtype = "Article", author = as.character(author),
            title = as.character(title),
            journaltitle = as.character(journal), year, key = row_number()) %>% 
  RefManageR::as.BibEntry() %>% 
  RefManageR::WriteBib("egeulgen.bib")

bib_entries <- bibliography_entries("egeulgen.bib")
bib_entries <- bib_entries[order(bib_entries$issued, decreasing = TRUE), ]
bib_entries
```
